md/customize-resource-layouts.md

```diff
@@ -0,0 +1,113 @@
+# Customize the resources layouts
```

Comment:
customize application layouts seems like a better name to me

Reply:
done